frontend/src/scenes/ingestion/CardContainer.tsx

```diff
@@ -19,45 +20,55 @@ export function CardContainer({
     showFooter?: boolean
     onSubmit?: () => void
 }): JSX.Element {
-    const { onboarding1 } = useValues(ingestionLogic)
+    const { onboarding1, onboardingSidebar } = useValues(ingestionLogic)
 
     return (
-        <div className="ingestion-card-container">
-            <Row align="middle" data-attr="wizard-step-counter">
-                {index !== 0 && (
-                    <ArrowLeftOutlined
-                        className="button-border clickable"
-                        style={{ marginRight: 4 }}
-                        onClick={onBack}
-                    />
+        <>
+            <div className="ingestion-card-container">
+                {onboardingSidebar ? (
+                    <>
+                        {index !== 0 && (
+                            <LemonButton className="mb" icon={<ArrowLeftOutlined />} size="small" onClick={onBack} />
```

Comment:
In Figam, it looks like the back button is a link above the container.

Reply:
it looks unnoticeable and kind of out of place there :( 

<img width="850" alt="Screen Shot 2022-05-20 at 12 48 41 PM" src="https://user-images.githubusercontent.com/25164963/169574906-31b00a8e-3871-4336-aab7-170d4f5f8783.png">

Reply:
i'll put it above the container just to align with mocks